apps/web/components/dialog/ReassignDialog.tsx

```diff
@@ -150,6 +153,24 @@ export const ReassignDialog = ({ isOpenDialog, setIsOpenDialog, teamId, bookingI
                 onChange={(event) => {
                   form.setValue("teamMemberId", event?.value);
                 }}
+                components={{
+                  Option: ({ children, ...props }) => {
+                    const status = props.data?.status;
```

Comment:
Override the current options component to provide a way to pass a status dot into the select